<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="deletebutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kembalibutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kembali</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kedisplinan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Produktivitas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Loyalitas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Masa Kerja</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kinerja</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rekomendasi</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="refreshbutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reportbutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="reportlabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
